From 3b6ebfec1b72148c9628525f6ebe2e6d184960b1 Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Fri, 6 Dec 2013 22:20:57 +0100
Subject: [PATCH 11/11] BUILD: proto_tcp: remove a harmless warning

The "pol" variable was not used and gcc 4.7 emits a warning on it.
---
 src/proto_tcp.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/proto_tcp.c b/src/proto_tcp.c
index 0158a02..d279669 100644
--- a/src/proto_tcp.c
+++ b/src/proto_tcp.c
@@ -870,7 +870,6 @@ static int tcp_parse_tcp_req(char **args, int section_type, struct proxy *curpx,
 	if (!strcmp(args[1], "content")) {
 		int action;
 		int warn = 0;
-		int pol = ACL_COND_NONE;
 		struct acl_cond *cond;
 		struct tcp_rule *rule;
 
@@ -891,7 +890,6 @@ static int tcp_parse_tcp_req(char **args, int section_type, struct proxy *curpx,
 			return -1;
 		}
 
-		pol = ACL_COND_NONE;
 		cond = NULL;
 
 		if (strcmp(args[3], "if") == 0 || strcmp(args[3], "unless") == 0) {
-- 
1.8.1.5

